common/djangoapps/util/milestones_helpers.py

```diff
@@ -411,5 +409,13 @@ def add_user_milestone(user, milestone):
     """
     if not settings.FEATURES.get('MILESTONES_APP', False):
         return None
-    from milestones import api as milestones_api
     return milestones_api.add_user_milestone(user, milestone)
+
+
+def remove_user_milestone(user, milestone):
+    """
+    Client API operation adapter/wrapper
+    """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
```

Comment:
nit: `, False` is not necessary.  When `.get('MILESTONES_APP')` without a second argument fails to find its key, it returns `None`, which is false enough.